render/helpers.go

```diff
@@ -95,3 +133,31 @@ func Markdown(md string) template.HTML {
 	s := blackfriday.Run([]byte(fmt.Sprintf("%s", modifiedMarkdown.String())))
 	return template.HTML(s)
 }
+
+// Localise localises text based on a key
+func Localise(key string, language string, plural int, templateArguments ...string) string {
+	if key == "" {
+		errorMessage := "key " + key + " not found in locale file"
+		log.Error(errors.New(errorMessage), nil)
```

Comment:
Logic perfectly fine but you can use `fmt.Errorf(....)` which would be the Go idiomatic way to do this. I won't object if you don't make this change.